src/main/java/org/folio/fqm/service/ResultSetService.java

```diff
@@ -33,7 +33,7 @@ private static List<Map<String, Object>> getSortedContents(List<UUID> contentIds
       .collect(Collectors.toMap(content -> (UUID) content.get(ID_FIELD_NAME), Function.identity()));
 
     return contentIds.stream()
-      .map(contentsMap::get)
+      .map(id -> contentsMap.getOrDefault(id, Map.of("id", id, "_deleted", true)))
```

Comment:
Small thing, but I'd move this `Map.of()` call outside the lambda. Otherwise, we end up allocating/populating an identical `Map` for every single row

Reply:
We'd have to do it here because the id would make each map unique right?

Reply:
Oh yeah, good point 😆